tests/test_connector.py

```diff
@@ -0,0 +1,66 @@
+from logging import getLogger
+from unittest import mock
+
+from pytest import fixture, fail
+
+import code42_connector
+from phantom.action_result import ActionResult
+import phantom.app as phantom
+import py42.sdk
+from requests import HTTPError
+
+
+logger = getLogger(name=__name__)
+
+
+NULL_VALUE = object()
+
+
+@fixture
+def py42_sdk(mocker):
+    return mocker.MagicMock(spec=py42.sdk)
+
+
+@fixture
+def py42_client(mocker):
+    return mocker.MagicMock(spec=py42.sdk.SDKClient)
+
+
+@fixture
+def connector(request, py42_client):
+    connector = code42_connector.Code42Connector()
+    connector._client = py42_client
+    yield connector
```

Comment:
is there a reason this is `yield` and not `return`?

Reply:
Think this was some copy+pasta from trying out the code generation feature of the new Phantom dev tooling. Removed.